src/doc/cli_manual/functions.rst

```diff
@@ -4605,10 +4551,7 @@ GetSelectionList
   GetSelectionList() -> SelectionList object
 
 
-none : type
-    
-
-return value : integer
+return type : SelectionList object
```

Comment:
When we have CLI methods for which the return type is an attribute object, any chance we can generate link-able references here to the associated attributes entry? Its easier to do this now than after the switch-over. It would involve two parts
1. Ensure that each attribute object defines its own, unique anchor
2. Geneerate the proper `.rst` reference wherever that object's name is used.

Might be too involved but would be nice.

Reply:
I think this is possible. The same script creates the functions and attributes pages, and it all pulls from the information in dir(visit). It's oo enough that this should be as easy as adding a field to the attribute and function classes and slightly modifying the methods that generate the rst.

I wonder how maintainable this is going forward where functions.rst becomes the file we edit for the cli documentation. It will still be easy to do step 1. of generating the anchors in the attributes (because attributes, I assume, will be generated from functions.rst). But now the developer is responsible for manually adding that link. It is conceivable that we should have a "linking" script that runs over the functions.rst and attributes.rst and modifies functions.rst to add links for these cases after the developer has modified functions.rst. So there might be some ways to do this that we can explore in the next phase.

Reply:
By "in the next phase" I am specifically referring to the linkage between *new* functions and attributes. I will update the script now to do this for the current functions and attributes.

Reply:
Ok, I like your idea of having a *linking script* to make a pass to do this. So, forget about this in the current phase now. In the GUI docs, we *did* manually do a lot of that kind of linking and it is indeed a pain but I was thinking it was only a pain *initially* got get into `.rst` in the first place. Once there, adding or revising that link data wouldn't be too bad. OTOH, if we can automate it, thats way better.